Parser/Data/graph.html

```diff
@@ -1,8 +1,10 @@
 <!DOCTYPE html>
-<head><meta charset="utf-8"></head>
+<head>
+    <meta charset="utf-8">
+</head>
 <body>
-  Graph of (3 + 2) * 4 - 17 * -five * (two + 1 + 4 + 5)
-  <pre class="mermaid">
+    Graph of (3 + 2) * 4 - 17 * -five * (two + 1 + 4 + 5)
+    <pre class="mermaid">

```

Comment:
Additional spaces are included here, which is not a part of code fix. Check before committing. These changes will be caught when doing a diff.